benchmarks/nel/scripts/candidate_generation/base.py

```diff
@@ -5,9 +5,10 @@
 
 import spacy
 from spacy import Language
-from spacy.kb import KnowledgeBase, Candidate
+from spacy.kb import Candidate
 from spacy.tokens import Span
 
+from compat import KnowledgeBase
```

Comment:
It would be more clear to usee relative imports like `.compat` - or was there a reason that didn't work somehow?

Reply:
Yes, relative imports failed despite `__init__.py` files being present. Relative imports in Python are my bane, but one of these days I'll figure it out for good :crossed_fingers: